dynatrace-aws-logs.sh

```diff
@@ -88,6 +91,23 @@ arguments:
     if [ -z $2 ] || [[ $2 == "--"* ]]; then echo "Missing value for parameter $1"; print_help_deploy; exit 1; fi
   }
 
+  check_api_token() {
+    if RESPONSE=$(curl -k -s -X POST -d "{\"token\":\"$TARGET_API_TOKEN\"}" "$TARGET_URL/api/v2/apiTokens/lookup" -w "<<HTTP_CODE>>%{http_code}" -H "accept: application/json; charset=utf-8" -H "Content-Type: application/json; charset=utf-8" -H "Authorization: Api-Token $TARGET_API_TOKEN"); then
+      CODE=$(sed -rn 's/.*<<HTTP_CODE>>(.*)$/\1/p' <<<"$RESPONSE")
+      RESPONSE=$(sed -r 's/(.*)<<HTTP_CODE>>.*$/\1/' <<<"$RESPONSE")
+      if [ "$CODE" -ge 300 ]; then
+        echo "Failed to check Dynatrace API token permissions - please verify provided values for parameters: --target-url and --target-api-token. $RESPONSE"
+        exit 1
+      fi
+      if ! grep -q "logs.ingest" <<<"$RESPONSE"; then
+        echo "Missing Ingest logs permission for the API token"
```

Comment:
I think you have recently been discussing to info if permissions are v1 or v2. I'd have it here as well (v2)